operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -505,6 +505,14 @@ rbac:
     seccompProfile:
       type: RuntimeDefault
 
+# -- KEDA Autoscaling configuration
+# Can be overridden by individual components configuration
```

Comment:
```suggestion
# -- KEDA Autoscaling configuration.
# Can be overridden by individual components configuration. For example distributor.kedaAutoscaling.
```


The dot is because this may get rendered as markdown and a single new line will just join the two lines